federation/external/allow.pub/config.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 The Sigstore Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+url: https://allow.pub
+contact: evan@phx.io
+description: "Server side signing support for the OCI registry vcr.pub"
```

Comment:
Neither vcr.pub or allow.pub resolve to anything meaningful in the browser.  Perhaps include (comment?) a way to learn more?

Reply:
Certainly! The websites are absolutely blank atm because I'm just getting those services up and going.

vcr.pub is the focus here, it's a simple, credit based OCI registry. Credit based meaning it's a bit like tarsnap where you pay upfront for credits and usage deducts from there. The idea is to build the best OCI registry that I want to use, which includes image signing, thus my interest in sigstore/fulcio.

vcr.pub is in alpha atm but I'd like to get the automated signing going before promoting it more, since this sort of integration is a great feature to talk about.

More than happy to answer any questions about it or anything else!

Reply:
You make a great point, since I'm talking about these things outside my own tiny circle I should go ahead and at least provide something meaningful for browsers. I'll get that sorted out this weekend.

Reply:
I've at least provided some redirects at https://allow.pub and https://vcr.pub for now.